start_test.sh

```diff
@@ -236,25 +236,20 @@ echo "slave_array=(${slave_array[@]}); index=${slave_num} && while [ \${index} -
     echo "echo \"Installing needed plugins for master\""
     echo "cd /opt/jmeter/apache-jmeter/bin" 
     echo "sh PluginsManagerCMD.sh install-for-jmx ${jmx}" 
-    echo "echo \"Done installing plugins, launching test\""
     echo "jmeter ${param_host} ${param_user} ${report_command_line} --logfile /report/${jmx}_$(date +"%F_%H%M%S").jtl --nongui --testfile ${jmx} -Dserver.rmi.ssl.disable=true --remoteexit --remotestart ${slave_list} >> jmeter-master.out 2>> jmeter-master.err &"
     echo "trap 'kill -10 1' EXIT INT TERM"
     echo "java -jar /opt/jmeter/apache-jmeter/lib/jolokia-java-agent.jar start JMeter >> jmeter-master.out 2>> jmeter-master.err"
-    echo "echo \"Starting load test at : $(date)\" && wait"
+    echo "wait"
 } >> "scenario/${jmx_dir}/load_test.sh"
 
-logit "INFO" "Copying scenario/${jmx_dir}/load_test.sh into  ${master_pod}:/opt/jmeter/load_test"
-kubectl cp -c jmmaster "scenario/${jmx_dir}/load_test.sh" -n "${namespace}" "${master_pod}:/opt/jmeter/load_test"
+logit "INFO" "Copying scenario/${jmx_dir}/load_test.sh into  ${master_pod}://opt/jmeter/load_test"
+kubectl cp -c jmmaster "scenario/${jmx_dir}/load_test.sh" -n "${namespace}" "${master_pod}://opt/jmeter/load_test"
 
 logit "INFO" "Starting the performance test"
-logit "INFO" "##################################################"
-logit "INFO" "You can follow test execution summary on the master pod by running :"
-logit "INFO" "         kubectl logs -f -c jmmaster -n ${namespace} ${master_pod}"
-logit "INFO" "##################################################"
-logit "INFO" "Also using Grafana : kubectl ${namespace} port-forward svc/grafana 8443:443"
-GRAFANA_LOGIN=$(kubectl -n "${namespace}" get secret grafana-creds -o yaml | grep GF_SECURITY_ADMIN_USER: | awk -F" " '{print $2}' | base64 --decode)
-GRAFANA_PASSWORD=$(kubectl -n "${namespace}" get secret grafana-creds -o yaml | grep GF_SECURITY_ADMIN_PASSWORD: | awk -F" " '{print $2}' | base64 --decode)
-logit "INFO" " LOGIN : ${GRAFANA_LOGIN}"
-logit "INFO" " PASSWORD : ${GRAFANA_PASSWORD}"
-logit "INFO" "################################################"
-
+logit "INFO" "${namespace} ${master_pod}"
+kubectl port-forward $(kubectl get pod | grep grafana |awk '{print $1}') 3000 &
+logit "INFO" "Connect on localhost:3000"
+logit "WARNING" "DO NOT SHARE THESES CREDENTIALS"
+logit "WARNING" "username: admin"
+logit "WARNING" "password: XhXUdmQ576H6e7"
```

Comment:
The warning about sharing the credentials is a good addition, thanks.

But why removing the dynamically retrieval of the grafana creds ? If somebody want to change the grafana creds (like using external secrets to avoid having there in clear text in the repo), it will not be user friendly

Reply:
I didn't meant to remove the dynamic retrieval of grafana creds.
It was just a help line for users to have the default creds when deploying the stack

Reply:
Using the dynamic retrieval, would the credentials be the default ones or the modified one, the user would get the most up to date value.